scripts/pre_commit_linter.py

```diff
@@ -3201,4 +3379,8 @@ def main():
 
 
 if __name__ == '__main__':
+    NAME_SPACE = multiprocessing.Manager().Namespace()
+    PROCESSES = multiprocessing.Manager().dict()
+    NAME_SPACE.files = FileCache()
+    FILE_CACHE = NAME_SPACE.files
```

Comment:
Can you move these inside `main` or does it not work?

Reply:
Those are intended to be global variables so I can not move them into `main`. I was trying to put them at the top of the file (e.g.: at about line 601), but it is using FileCache which has not been defined at this line yet. So I moved it to this position.

Reply:
Okay, that's fine.